radio/src/telemetry/frsky.h

```diff
@@ -139,8 +139,10 @@
 #define RBOX_STATE_LAST_ID        0x0b2f
 #define RBOX_CNSP_FIRST_ID        0x0b30
 #define RBOX_CNSP_LAST_ID         0x0b3f
-#define DIY_FIRST_ID              0x5000
+#define DIY_FIRST_ID              0x5100
```

Comment:
Yes this means that 5000-52ff are free to use, BUT we don't want all of that to go to lua exclusively. 
We want 5000-50ff to go to lua, and the rest to be available for standard sensors.